core/parentchain/light-client/src/light_validation.rs

```diff
@@ -160,44 +151,6 @@ where
 		self.submit_finalized_headers(header.clone(), vec![], justifications)
 	}
 
-	fn check_xt_inclusion(&mut self, block: &Block) -> Result<(), Error> {
-		let relay = self.light_validation_state.get_relay_mut();
-
-		if relay.verify_tx_inclusion.is_empty() {
-			return Ok(())
-		}
-
-		let mut found_xts = vec![];
-		block.extrinsics().iter().for_each(|xt| {
-			if let Some(index) = relay.verify_tx_inclusion.iter().position(|xt_opaque| {
-				<HashingFor<Block>>::hash_of(xt) == <HashingFor<Block>>::hash_of(xt_opaque)
-			}) {
-				found_xts.push(index);
-			}
-		});
```

Comment:
This was the piece of code that didn't work and that caused a delay in block import.